tests/numerics/test_time_step_control.py

```diff
@@ -724,3 +748,139 @@ def test_io_time_information(self):
         )
         # Remove temporary file.
         pth.unlink()
+
+
+class DynamicTimeStepTestCaseModel(SquareDomainOrthogonalFractures, SinglePhaseFlow):
```

Comment:
Nice mock-class! Any reason to use a domain with fractures? If yes, please make sure the grid is Cartesian and coarse (this may actually save a few tens of a seconds for every run of the tests).

Reply:
I agree, it can be removed